DirectProgramming/C++SYCL_FPGA/Tutorials/Features/experimental/hostpipes/README.md

```diff
@@ -328,10 +300,6 @@ In the latter launch-collect test, the entire contents of the `in` vector are wr
       ```
       make fpga_sim
       ```
-   4. Compile and run on FPGA hardware (longer compile time, targets an FPGA device).
-      ```
-      make fpga
```

Comment:
Why removing the make fpga target? One can still compile using Quartus to get accurate resource usage / fmax reports